pkg/operator/client/client.go

```diff
@@ -38,6 +39,9 @@ func init() {
 	if err := iov1.AddToScheme(scheme); err != nil {
 		panic(err)
 	}
+	if err := routev1.Install(scheme); err != nil {
```

Comment:
Kind of weird that we need to import the route API in `pkg/operator` in order to use it in the test code.  Because the scheme is registered at run-time (even if it is never used), it gets linked into the operator binary, right?

Reply:
I could probably have done it in a test file, but given https://github.com/openshift/cluster-ingress-operator/pull/345#discussion_r366550997 I'll probably just end up deleting this in the e2e-relocation followup if that's the direction we go.